docker/windows/Scripts/configure-tentacle.ps1

```diff
@@ -254,7 +258,7 @@ function Register-Tentacle(){
     $arg += "`"$Space`"";
   }
 
-  Execute-Command $TentacleExe $arg;
+  Execute-Command $TentacleExe $arg $mask;
```

Comment:
This calls https://github.com/OctopusDeploy/OctopusTentacle/blob/56ab573fc4b8217b6ea959e3f10eed8740b1ad62/docker/windows/Scripts/octopus-common.ps1#L19 which does the masking